src/main/java/net/javadiscord/javabot/systems/commands/ProfileCommand.java

```diff
@@ -64,6 +66,7 @@ private MessageEmbed buildProfileEmbed(Member member, Connection con) throws SQL
 						new LeaderboardCommand().getQOTWRank(member, member.getGuild())), true)
 				.addField("Server joined", String.format("<t:%s:R>", member.getTimeJoined().toEpochSecond()), true)
 				.addField("Account created", String.format("<t:%s:R>", member.getUser().getTimeCreated().toEpochSecond()), true);
+		embed.addField("Total Help XP", String.format("%.2f XP", helpXP), true);
```

Comment:
```suggestion
						new LeaderboardCommand().getQOTWRank(member, member.getGuild())), true)
				.addField("Total Help XP", String.format("%.2f XP", helpXP), true),
				.addField("Server joined", String.format("<t:%s:R>", member.getTimeJoined().toEpochSecond()), true)
				.addField("Account created", String.format("<t:%s:R>", member.getUser().getTimeCreated().toEpochSecond()), true);

```
I think this should be moved up here to not put information like this after the dates

Reply:
Makes sense